src/main/docker/jenkins/security.groovy

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.security.*
+import jenkins.security.s2m.AdminWhitelistRule
+
+def instance = Jenkins.getInstance()
+
+def hudsonRealm = new HudsonPrivateSecurityRealm(false)
+hudsonRealm.createAccount("admin", "admin")
+instance.setSecurityRealm(hudsonRealm)
+
+def strategy = new FullControlOnceLoggedInAuthorizationStrategy()
```

Comment:
Could you document why we need to change the default behavior?

Reply:
No idea, look at this documentation here https://javadoc.jenkins-ci.org/jenkins/model/Jenkins.html#setAuthorizationStrategy-hudson.security.AuthorizationStrategy-
As you can see it's non existant.
It seems like we perhaps need to set a strategy if we overwrite the instance security.

Reply:
Apparently no security is default so we add startegy that gives logged in user full control. I've added some comments at the top of the file.